cmp/leftrightchooser/LeftRightChooserFilter.js

```diff
@@ -49,17 +51,21 @@ class LeftRightChooserFilter extends Component {
     }
     
     runFilter() {
-        const fields = this.props.fields || [],
+        const {fields = [], anyMatch} = this.props,
```

Comment:
I know this is not new code, but what would the point be of having an empty fields array? That seems like it's an invalid usage.

Would suggest we mark the array as required in the propTypes - is there a way to specify required + non-empty for arrays?